setup.py

```diff
@@ -136,6 +142,7 @@ def configuration(parent_package='', top_path=None):
         ],
         install_requires=INSTALL_REQUIRES,
         requires=REQUIRES,
+        python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```

Comment:
Are you sure we are not 3.{0,1,2,3} compatible? I don't think it's very important, but I don't want *unnecessary* restrictions here. Also iirc `python_requires` is a relatively recent addition?

Reply:
Python 3.0-3.3 are not tested in Travis CI or AppVeyor, so it's certainly quite possible they don't work.

They're also all no longer receiving security updates (or any updates) from the core Python team. The oldest was EOL 9 years ago. https://en.wikipedia.org/wiki/CPython#Version_history

And they're little used. Here's the pip installs for scikit-image from PyPI for August 2018:

| python_version | percent | download_count |
| -------------- | ------: | -------------: |
| 3.6            |  91.88% |      3,510,384 |
| 2.7            |   5.96% |        227,812 |
| 3.5            |   1.71% |         65,496 |
| 3.7            |   0.22% |          8,556 |
| 3.4            |   0.21% |          8,168 |
| 2.6            |   0.00% |              8 |
| 3.3            |   0.00% |              7 |
| 3.8            |   0.00% |              2 |
| 3.2            |   0.00% |              1 |
| Total          |         |      3,820,434 |

Source: `pypinfo --start-date 2018-08-01 --end-date 2018-08-31 --percent --markdown scikit-image pyversion`

(Wow, look at all that 3.6! 🎆)

But to be on the safe side, would you prefer `python_requires='>=2.7'`?

Reply:
✅